test/assembly/GraknRunner.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2021 Grakn Labs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package grakn.common.test.assembly;
+
+import org.apache.commons.io.FileUtils;
+import org.zeroturnaround.exec.ProcessExecutor;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.TimeoutException;
+
+import static org.junit.Assert.fail;
+
+public abstract class GraknRunner {
```

Comment:
Let's call it `Runner` since it is for running not just `Grakn` products but also `Console`

Reply:
Isn't `GraknConsole` the official product name of the console? At least that's the name of the distribution files.

Reply:
actually it dosn't matter, I will change the name.

Reply:
In the docs I think we also refer to the database as Grakn Core / Grakn Cluster, but for the interface, Console and Workbase rather than Grakn Console and Grakn Workbase. Also the repo is called `console` and `workbase`.

These are my anchors for the reasoning.